RNUnifiedContacts/RNUnifiedContacts.swift

```diff
@@ -163,17 +163,94 @@ class RNUnifiedContacts: NSObject {
 
       callback([error.localizedDescription, NSNull()])
     }
-
   }
-
-
-
-
-
+  
+  @objc func createContact(contactData: NSDictionary, callback: (NSObject) -> () ) -> Void {
```

Comment:
Also, since the iOS version is called `addContact`, what do you think about calling this `addContact` instead, just to mirror iOS as much as possible?